### ISSUE TITLE
Trouble finding shared objects when running hello_classification

### ISSUE THREAD

**Issue by mjhammel** [openvinotoolkit/dldt#460]
Here is a sample run and proof that the missing object is installed (but maybe in the wrong place).

----

$ hello_classification /usr/share/openvino/face-detection-adas-0001.xml /usr/share/openvino/images/engineers-in-meeting-3912477.jpg GPU
unable to load /opt/test/qsys/os/x86_64-linux-gnu/beignet-install/usr/lib/beignet//libgbeinterp.so which is part of the driver, please check!
Failed to create plugin /usr/lib64/libclDNNPlugin.so for device GPU
Please, check your environment
[CLDNN ERROR]. No GPU device was found.

$ ls -l /usr/lib/beignet/
total 445
-rwxr-xr-x    1 root     root        364176 Apr 15 18:41 libcl.so
-rwxr-xr-x    1 root     root         90544 Apr 15 18:41 libgbeinterp.so

$ echo $LD_LIBRARY_PATH
/usr/lib/beignet:

-------------------------------------

It looks like I compiled things incorrectly, perhaps?  I'm building dldt from source but using the binary distribution of compute module.  Why is it looking in a specific path for the object?  strace says that's the only place it looks for that shared object.  Is that path configurable with an environment variable, perhaps?

**Comment by ilya-lavrenov** [comment]
Looks like something is wrong with `/opt/test/qsys/os/x86_64-linux-gnu/beignet-install/usr/lib/beignet//libgbeinterp.so` on your system. Is this library on your system? Does `clinfo` work properly?

**Comment by dkurt** [comment]
Beignet has been deprecated long time ago (https://01.org/beignet). You need to install computw runtime instead (NEO) - https://github.com/intel/compute-runtime

**Comment by mjhammel** [comment]
I found the problem, I think.  There is a legacy installation of OpenCL on the system already.  That instance looks for /etc/OpenCL/vendors and that's why it's looking for beignet.  There are two ways to tell OpenCL to look somewhere else.  One is to set OPENCL_VENDOR_PATH, but that didn't work with this version of OpenCL.  Instead, I had to use OCL_ICD_VENDORS (which I found by running strings on the OpenCL library).  Now when I run the sample program I don't get the error about beignet.  The sample program runs okay with CPU now but still doesn't work with GPU.

I now have another question about whether my Atom is supported or not but I'll ask that separately and close this issue.
